src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining.Api/Controllers/EmployerRequestsController.cs

```diff
@@ -109,12 +108,16 @@ public async Task<IActionResult> GetSelectEmployerRequests(string standardRefere
             }
         }
 
-        [HttpPost("provider/responses")]
-        public async Task<IActionResult> CreateProviderResponse(CreateProviderResponseEmployerRequestCommand command)
+        [HttpPost("provider/{ukprn}/acknowledge-requests")]
+        public async Task<IActionResult> CreateProviderResponse(long ukprn, AcknowledgeRequestsParameters parameters)
```

Comment:
Need to rename the method and the command etc to match
AcknowledgeEmployerRequests and AcknowledgeEmployerRequestsCommand